javatests/arcs/android/integration/policy/PolicyTest.kt

```diff
@@ -80,27 +82,90 @@ class PolicyTest {
     assertOnlySingletonAbIsPersisted(startingKey)
   }
 
-  private suspend fun assertOnlySingletonAbIsPersisted(startingKey: StorageKey) {
+  /**
+   * Scenario: A policy-compliant recipe with @ttl handles egresses data and stores
+   * ingress-restricted values to RAM that are deleted after the TTL expires and at the runtime’s
+   * end.
+   */
+  @Test
+  fun ttlHandlesWithEgress_StoresIngressRestrictedValues_deletedAtTtlTimeAndRuntimeEnd() =
+    runBlocking {
```

Comment:
Is `runBlocking` OK in tests?

Reply:
Yes, this is the standard from what I can tell.